### PR TITLE
se agrega el R Dev Day

### PR DESCRIPTION
**Pull request by PatriLoto**
se agregaron los archivos correspondientes al evento R Dev Day

### CODE REVIEW

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+**Modo:** Online
+
+**Inscrição:** <https://pretix.eu/r-contributors/r-dev-day-latinr2024/>
+
+**Idiomas:** Inglês, português e espanhol
+
+# Sobre
+
+O R Dev Day \@ LatinR 2024 é um evento colaborativo no qual os participantes trabalharão em pequenos grupos para contribuir para o R básico ou para a infraestrutura que dá suporte a essas contribuições, por exemplo
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
O R Dev Day \@ LatinR 2024 é um evento colaborativo no qual as pessoas participantes trabalharão em pequenos grupos para contribuir para o R base ou para a infraestrutura que dá suporte a essas contribuições, por exemplo:
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+-   Traduzir strings de mensagens para espanhol/português brasileiro
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
-   Traduzir textos de mensagens para espanhol/português brasileiro
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+-   Propor melhorias nos arquivos de ajuda do R
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
-   Propor melhorias nos arquivos de ajuda do R (conteúdo que acessamos no `help` ou `?` no R)
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+-   Analisar um erro relatado no R
+-   Desenvolver uma correção para um erro relatado no R
+-   Adicionar conteúdo ao [R Dev Guide](https://contributor.r-project.org/rdevguide/)
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
-   Adicionar conteúdo ao [R Dev Guide](https://contributor.r-project.org/rdevguide/), o guia de contribuição ao R base
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+-   Melhorar o [R Translations Dashboard](https://contributor.r-project.org/translations-dashboard/)
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
-   Melhorar o [R Translations Dashboard](https://contributor.r-project.org/translations-dashboard/), o painel utilizado para acompanhar o andamento das traduções 
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+Tanto colaboradores novos quanto experientes estão convidados a participar!
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
Tanto pessoas colaboradoras novas quanto experientes estão convidadas a participar!
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+## Quando
+
+Haverá duas sessões na segunda-feira, 18 de novembro, e na terça-feira, 19 de novembro. No mesmo horário todos os dias:
+
+| UTC           | Buenos Aires (UTC-3) | Hermosillo (UTC-7) |
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
| UTC           | Buenos Aires e Brasília (UTC-3) | Hermosillo (UTC-7) |
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+Recomendamos que você participe de ambas as sessões, se possível, mas não há problema em participar de apenas uma sessão.
+
+## Comunicação
+
+Haverá uma breve introdução (\~10 minutos) no início de cada sessão em inglês, espanhol e português.
+
+Usaremos o canal "r-dev-day" no Slack do LatinR para discussões relacionadas - o uso de inglês, espanhol ou português é bem-vindo.
+
+Os pequenos grupos podem acordar um idioma comum para trabalhar - o inglês será o padrão, mas os facilitadores que falam espanhol e português estarão disponíveis para ajudar.
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
Os grupos menores podem acordar um idioma comum para trabalhar - o inglês será o padrão, mas as pessoas facilitadoras que falam espanhol e português estarão disponíveis para ajudar.
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+## Tarefas
+
+As tarefas serão preparadas como [problemas neste repositório com o rótulo `LatinR2024`](https://github.com/r-devel/r-dev-day/issues?q=is%3Aissue+is%3Aopen+label%3ALatinR2024). Você pode adicionar problemas ou discutir ideias para problemas, conforme descrito no [README](https://github.com/r-devel/r-dev-day/blob/main/README.md).
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
As tarefas serão preparadas como [*issues* neste repositório com o rótulo `LatinR2024`](https://github.com/r-devel/r-dev-day/issues?q=is%3Aissue+is%3Aopen+label%3ALatinR2024). Você pode adicionar *issues* ou discutir ideias para *issues*, conforme descrito no [README](https://github.com/r-devel/r-dev-day/blob/main/README.md).
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+Os problemas estão resumidos na planilha do [R Dev Day \@ LatinR 2024 Working Group Allocations](https://docs.google.com/spreadsheets/d/1Nuwusm7Xgwpw54KZnnrnNVT9vPkOk2qmromBEbYNHnU/edit). Adicione seu nome à lista de colaboradores se estiver interessado em trabalhar em um problema. Você pode se incluir em várias questões, pois não se trata de um compromisso final - as alocações serão finalizadas no início de cada sessão.
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
As *issues* estão resumidos na planilha do [R Dev Day \@ LatinR 2024 Working Group Allocations](https://docs.google.com/spreadsheets/d/1Nuwusm7Xgwpw54KZnnrnNVT9vPkOk2qmromBEbYNHnU/edit). Adicione seu nome à lista de pessoas colaboradoras se tiver interesse em trabalhar em uma *issue*. Você pode se incluir em várias *issues*, pois não se trata de um compromisso final - as alocações serão finalizadas no início de cada sessão.
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+
+Fique de olho nas edições antes do evento. Talvez você precise de preparação adicional para trabalhar em determinadas questões, por exemplo, aprender a criar o R no macOS ou ler um material de apoio.
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
Fique de olho nas edições antes do evento. Talvez você precise de preparação adicional para trabalhar em determinadas questões, por exemplo, aprender a fazer o *build* do R no macOS ou ler um material de apoio.
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+
+## Configuração do desenvolvimento
+
+A maioria das pessoas poderá trabalhar somente pelo navegador (usando o Weblate, a interface do navegador do GitHub, o GitHub Codespaces ou o Gitpod). O trabalho com bugs específicos do sistema operacional pode exigir o uso ou a criação do R em seu próprio computador.
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
A maioria das pessoas poderá trabalhar somente pelo navegador (usando o Weblate, a interface do navegador do GitHub, o GitHub Codespaces ou o Gitpod). O trabalho com bugs específicos do sistema operacional pode exigir fazer o *build* do R no seu próprio computador.
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+
+## Preparação
+
+Algumas coisas que você pode fazer para ajudar a se preparar, se tiver tempo:
+
+-   [Solicite uma conta no Bugzilla](https://contributor.r-project.org/rdevguide/IssueTrack.html#bugzilla-account) do R se você quiser trabalhar em bugs no código/documentação e ainda não tiver uma conta. O processo pode levar alguns dias, mas é apenas um "é bom ter".
+-   [Crie uma conta no Weblate](https://translate.rx.studio/projects/r-project/) se você quiser trabalhar com traduções. Isso também pode ser feito durante o evento.
+-   Se quiser trabalhar em bugs no código do R (em comparação com a documentação, o R Translations Dashboard ou o R Dev Guide) e não se sentir confortável com a criação do R a partir do código-fonte, recomendamos que aprenda a usar o R Dev Container, conforme descrito na seção [Outros recursos](#outros-recursos) (leva aproximadamente 1 hora). Isso não será necessário para todas as tarefas que envolvem código, mas abrirá mais possibilidades.
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
-   Se quiser trabalhar em bugs no código do R (em comparação com a documentação, o *R Translations Dashboard* ou o *R Dev Guide*) e não se sentir confortável com fazer o *build* do R a partir do código-fonte, recomendamos que aprenda a usar o *R Dev Container*, conforme descrito na seção [Outros recursos](#outros-recursos) (leva aproximadamente 1 hora). Isso não será necessário para todas as tarefas que envolvem código, mas abrirá mais possibilidades.
```

File: cronograma/r_dev_day/RDevDay-pt.qmd [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+
+[Starting Workspace](https://contributor.r-project.org/r-dev-env/container_setup/gitpod_workspace/workspacestart/) e [Stopping and restarting](https://contributor.r-project.org/r-dev-env/container_setup/gitpod_workspace/workspacestop_and_restart/), para aprender a usar o R Dev Container por meio do navegador com espaços de trabalho do Gitpod, seguido por [Running R](https://contributor.r-project.org/r-dev-env/tutorials/running_r/), [Building R](https://contributor.r-project.org/r-dev-env/tutorials/building_r/) e [R Contribution Workflow](https://contributor.r-project.org/r-dev-env/tutorials/contribution_workflow/) para aprender os fundamentos do uso do R Dev Container para desenvolver alterações no R básico (guia passo a passo de \<1 hora).
+
+[R Dev Guide](https://contributor.r-project.org/rdevguide/) (ebook). Foi criado como uma referência para colaboradores sobre diferentes aspectos da contribuição para o R. Em particular, consulte o Capítulo 2 para saber como criar o R em seu próprio computador. Atualmente, as instruções para macOS não foram incorporadas, mas há uma [versão preliminar](https://github.com/r-devel/rcwg/blob/main/working_documents/install_r_macos.md).
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-pt.qmd]:
```suggestion
[R Dev Guide](https://contributor.r-project.org/rdevguide/) (ebook). Foi criado como uma referência para pessoas colaboradoras sobre diferentes aspectos da contribuição para o R. Em particular, consulte o Capítulo 2 para saber como fazer o *build* do R no seu próprio computador. Atualmente, as instruções para macOS não foram incorporadas, mas há uma [versão preliminar](https://github.com/r-devel/rcwg/blob/main/working_documents/install_r_macos.md).
```

**Review by beatrizmilz — COMMENTED** [review]
Hi Patri!
Thank you for submitting!
This week has been really busy, I'm travelling to a conference tomorrow. This conference appeared quite close, and I had to do a lot of tasks and meetings that was scheduled for next week, but this week.

I wrote some suggestions in the Portuguese version! Let me know if it is ok to accept them.
Thank you!

@hturner

File: cronograma/r_dev_day/RDevDay-pt.qmd (hunk 3 above)
Comment (hturner) [reply cronograma/r_dev_day/RDevDay-pt.qmd]: Apply similar update to English:

-   Propose improvements to the R help files (that are accessed via `help` or `?` in R)

File: cronograma/r_dev_day/RDevDay-pt.qmd (hunk 4 above)
Comment (hturner) [reply cronograma/r_dev_day/RDevDay-pt.qmd]: Apply similar update to English:

-   Add content to the [R Dev Guide](https://contributor.r-project.org/rdevguide/), the guide for contributors to base R

File: cronograma/r_dev_day/RDevDay-pt.qmd (hunk 5 above)
Comment (hturner) [reply cronograma/r_dev_day/RDevDay-pt.qmd]: Apply similar update to English:

- Improve the [R Translations Dashboard](https://contributor.r-project.org/translations-dashboard/), the dashboard used to monitor the status of translations

File: cronograma/r_dev_day/RDevDay-pt.qmd (hunk 7 above)
Comment (hturner) [reply cronograma/r_dev_day/RDevDay-pt.qmd]: Copy over to English and Spanish translation too

**Comment by hturner** [comment]
Thanks @beatrizmilz, I'm happy to go with your suggested updates on wording - where you have added further detail, we should apply this to the English and Spanish as well. I've provided the updates required for the English above, hopefully @PatriLoto can adjust the Spanish version accordingly.

File: cronograma/r_dev_day/index.qmd [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+-   Añadir contenido a la [Guía de Desarrollo de R](https://contributor.r-project.org/rdevguide/)
+-   Mejorar el [Tablero de traducciones de R](https://contributor.r-project.org/translations-dashboard/)
+
+Invitamos a participar a personas con distintos niveles de conocimiento, desde principiantes hasta expertos.
+
+## Cuándo
+
+Las sesiones se realizarán el lunes 18 y martes 19 de noviembre, a la misma hora cada día.
+
+| UTC           | Buenos Aires (UTC-3) | Hermosillo (UTC-7) |
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/index.qmd]:
```suggestion
| UTC           | Buenos Aires y Brasília (UTC-3) | Hermosillo (UTC-7) |
```

File: cronograma/r_dev_day/RDevDay-en.qmd [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+-   Adding content to the [R Dev Guide](https://contributor.r-project.org/rdevguide/)
+-   Improving the [R Translations Dashboard](https://contributor.r-project.org/translations-dashboard/)
+
+Both new and experienced contributors are encouraged to attend!
+
+## When
+
+There will be two sessions on Monday 18 November and Tuesday 19 November at the same time each day:
+
+| UTC           | Buenos Aires (UTC-3) | Hermosillo (UTC-7) |
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-en.qmd]:
```suggestion
| UTC           | Buenos Aires and Brasília (UTC-3) | Hermosillo (UTC-7) |
```

File: cronograma/r_dev_day/RDevDay-en.qmd [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+**Enrollment:** <https://pretix.eu/r-contributors/r-dev-day-latinr2024/>
+
+**Language:** English, Portuguese and Spanish
+
+## About
+
+R Dev Day \@ LatinR 2024 is a collaborative event where participants will work in small groups on contributions to base R, or to infrastructure that supports such contribution, e.g.,
+
+-   Translating message strings to Spanish/Brazilian Portuguese
+-   Proposing improvements to R help files
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-en.qmd]:
```suggestion
-   Proposing improvements to the R help files (that are accessed via `help` or `?` in R)
```

File: cronograma/r_dev_day/RDevDay-en.qmd [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+-   Analysing a reported bug in R
+-   Developing a fix to a reported bug in R
+-   Adding content to the [R Dev Guide](https://contributor.r-project.org/rdevguide/)
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-en.qmd]:
```suggestion
-   Adding content to the [R Dev Guide](https://contributor.r-project.org/rdevguide/), the guide for contributors to base R
```

File: cronograma/r_dev_day/RDevDay-en.qmd [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+-   Improving the [R Translations Dashboard](https://contributor.r-project.org/translations-dashboard/)
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/RDevDay-en.qmd]:
```suggestion
-   Improving the [R Translations Dashboard](https://contributor.r-project.org/translations-dashboard/), the dashboard used to monitor the status of translations
```

File: cronograma/r_dev_day/index.qmd [hunk 20]
```diff
@@ -0,0 +1,96 @@
...
+**Inscripción:** <https://pretix.eu/r-contributors/r-dev-day-latinr2024/>
+
+**Idioma:** Inglés, Portugués y Español
+
+## Acerca de
+
+R Dev Day \@ LatinR 2024 es un evento colaborativo donde las personas trabajarán en pequeños grupos para hacer contribuciones a R base, o a la infraestructura que soporta dicha contribución. Ejemplos de posibles contribuciones:
+
+-   Traducir mensajes de error cadenasal español/portugués de Brasil
+-   Proponer mejoras en la documentación de R
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/index.qmd]:
```suggestion
-   Proponer mejoras en la documentación de R (contenido que accedemos en `help` o `?` en R)
```

@PatriLoto please review 🙏

File: cronograma/r_dev_day/index.qmd [hunk 21]
```diff
@@ -0,0 +1,96 @@
...
+-   Analizar un error en R
+-   Generar una solución a un error en R
+-   Añadir contenido a la [Guía de Desarrollo de R](https://contributor.r-project.org/rdevguide/)
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/index.qmd]:
```suggestion
-   Añadir contenido a la [Guía de Desarrollo de R](https://contributor.r-project.org/rdevguide/), la guía de contribución para el R base
```
@PatriLoto please review 🙏

File: cronograma/r_dev_day/index.qmd [hunk 22]
```diff
@@ -0,0 +1,96 @@
...
+-   Mejorar el [Tablero de traducciones de R](https://contributor.r-project.org/translations-dashboard/)
```
Comment (beatrizmilz) [inline cronograma/r_dev_day/index.qmd]:
```suggestion
-   Mejorar el [Tablero de traducciones de R](https://contributor.r-project.org/translations-dashboard/), el panel utilizado para monitorear el estado de las traducciones
```

@PatriLoto please review 🙏

**Comment by PatriLoto** [comment]
Done, thank you Bea for the suggestions!

**Comment by PatriLoto** [comment]
@beatrizmilz, could you please merge the PR and add the link of the event in the 'Shedule' Menu??

**Comment by beatrizmilz** [comment]
Done! @PatriLoto @hturner 
https://latinr.org/cronograma/r-dev-day/
https://latinr.org/cronograma/r-dev-day/rdevday-pt.html
https://latinr.org/cronograma/r-dev-day/rdevday-en.html

**Comment by PatriLoto** [comment]
Thank you, Bea!

**Pull request merged**